modules/installing-gcp-user-defined-tags.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-tags_{context}"]
+= Configuring the user-defined tags for GCP
+
+
+In {product-title}, you can use the tags for grouping, organizing and for managing resources created for a particular {product-title} cluster.
```

Comment:
Could we please rephrase this line to mean, GCP provides tags functionality which can be used for grouping, organizing and for managing resources created for a particular {product-title} cluster.

Reply:
Done.